configure

```diff
@@ -446,6 +446,18 @@ parser.add_option('--with-external-spidermonkey-release',
     default='',
     help='enable linking against an out of tree SpiderMonkey release build')
 
+parser.add_option('--with-system-nspr',
+    action='store',
+    dest='system_nspr',
+    default='',
+    help='use system NSPR when linking against an out-of-tree SpiderMonkey')
```

Comment:
The Gecko flags serve a completely different purpose: they use a system library for NSPR and NSS when building Gecko instead of the in-tree ones.  However, here we need flags that control whether we add Gecko's NSPR and NSS libraries built in-tree as runtime dependencies.  They don't control whether the SpiderNode build system builds them (we never build them these days in the default config.)